cmd/nameset-backfill/main.go

```diff
@@ -71,18 +71,18 @@ func (b *backfiller) run() error {
 
 func (b *backfiller) findCerts() ([]resultHolder, error) {
 	var allResults []resultHolder
+	lastSerial := ""
 	for {
 		var results []resultHolder
 		_, err := b.dbMap.Select(
 			&results,
 			`SELECT c.serial, c.issued, c.expires, c.der FROM certificates AS c
        LEFT JOIN fqdnSets AS ns ON c.serial=ns.serial
        WHERE ns.serial IS NULL
-       ORDER BY c.issued DESC
-       LIMIT ?
-       OFFSET ?`,
+       AND c.serial > ?
```

Comment:
Here is a quick example of the natural sort property: https://gist.github.com/rolandshoemaker/236055902509383b8c873dda8a5c1da8